user/src/components/Login/Login.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
+import './Login.scss'
+import { Modal, Form, Button } from 'react-bootstrap'
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import validate from '../../features/validate';
+import { loginRequest } from '../../actions/auth.action';
+
+const Login = (props) => {
+    const { isOpen, handleCloseLogin } = props
```

Comment:
```suggestion
    const { isOpen, handleClose } = props
```
hoặc onClose là được rồi, k cần đặt dài quá đâu